### PR TITLE
Allow installing on eZ kernel 7.0

### PR DESCRIPTION
**Pull request by emodric**
Now sure what will happen, but let's see :)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "ezsystems/ezpublish-kernel": "^6.11@dev"
+        "ezsystems/ezpublish-kernel": "^6.11@dev|^7.0@dev"
```
Comment (alongosz) [inline composer.json]: Hmm, nitpick: recently I've found [in the composer doc](https://getcomposer.org/doc/articles/versions.md#version-range) that actually OR operator is `||`. Wasn't single pipe some old bc syntax? 
Just asking, as I've heard from Ivo, you're experienced in composer magic due to Sylius and eZ integrations ;)

Comment (emodric) [reply composer.json]: I have no problem using either :)

`composer validate` does not complain when I use `|`, so going by that, both are fine by me. But if you wish, I can change it to `||`.

Comment (alongosz) [reply composer.json]: yes, please :) I'm trying to enforce `||` whenever I see it :)
btw. nice tip with `composer validate`, somehow feels like basic tool that I haven't used :)

Comment (emodric) [reply composer.json]: Done!

**Pull request merged**